paloma_sdk/client/lcd/api/scheduler.py

```diff
@@ -0,0 +1,105 @@
+import json
+import base64
+from pathlib import Path
+from typing import Optional
+
+from ._base import BaseAsyncAPI, sync_bind
+from ..wallet import Wallet
+from .tx import CreateTxOptions
+from terra_proto.cosmwasm.wasm.v1 import AccessType
+from paloma_sdk.core.wasm.data import AccessConfig
+from paloma_sdk.core.wasm import MsgInstantiateContract, MsgExecuteContract, MsgStoreCode
+from paloma_sdk.core.scheduler import MsgCreateJob, MsgExecuteJob
+from paloma_sdk.core.coins import Coins
+from paloma_sdk.core.broadcast import BlockTxBroadcastResult
+from paloma_sdk.util.contract import read_file_as_b64
+
+__all__ = ["AsyncJobSchedulerAPI", "JobSchedulerAPI"]
+
+
+class AsyncJobSchedulerAPI(BaseAsyncAPI):
+    async def create_job(
+        self,
+        wallet: Wallet,
+        job_id: str,
+        contract_address: str,
+        abi: dict,
+        payload: str,
+        chain_type: str,
+        chain_reference_id: str
+    ) -> BlockTxBroadcastResult:
+        """create job
+        """
+        definition = {"abi":json.dumps(abi, separators=[",", ":"]),"address":contract_address}
+        definition_bytes = json.dumps(definition, separators=[",", ":"])
+        payload_bytes = json.dumps({"hexPayload": payload}, separators=[",", ":"])
```

Comment:
Neither one of these are bytes specifically. Possibly `definition_enc`/`payload_enc` (for encoded)?